source/uwp/Renderer/lib/pch.h

```diff
@@ -40,6 +40,10 @@
 #include "XamlBuilder.h"
 #include "XamlHelpers.h"
 
+namespace Helpers = ::AdaptiveCards::Rendering::Uwp;
```

Comment:
I wonder if there's a better naming we could use for this namespace as it may be confusing to read it besides ther XamlHelpers

Reply:
Do you have any suggestions for a better one? I already used "render_xaml" for winrt::AdaptiveCards::Rendering::Uwp, but this is a different namespace NOT in winrt. Maybe "rendering"? The lower-case r would differentiate it from the Rendering inside it.

Reply:
Alternatively, change the existing render_xaml to winrt_render_xaml and this to render_xaml.

Reply:
both `rendering `or changing render_xaml to winrt_render_xaml will work fine:)